BD-2316 Update SMS Double Opt-In (#5871)
braze-inc/braze-docs: _docs/_hidden/private_betas/sms_double_opt_in.md

REST API | By default, users won’t be enrolled when their subscription status is set by `/users/track` or any of the `subscription/status/set` endpoints. Users can be enrolled by passing an optional parameter `send_double_opt_in_response` as `true` (e.g., [{"subscription_group_id" : "subscription_group_identifier", "subscription_state" : "subscribed", "send_double_opt_in_response: true}]). 
Shopify | Users won’t be enrolled when their subscription status is set by our Shopify integration.
User Import | Users won’t be enrolled when their subscription status is set by User Import.
[Preference Center]({{site.baseurl}}/user_guide/message_building_by_channel/email/preference_center) | Users are automatically enrolled when subscribed through a preference center.
{: .reset-td-br-1 .reset-td-br-2}

## Multi-language support

> For inbound messages, SMS double opt-in is supported for all languages defined in the Subscription Group. This means users can define their auto-responses in different languages and Braze will send the auto-response associated with a specific language when a matching keyword is received.

Reviewer: This seems to mean Braze users, so I would recommend "you" here to speak directly to the reader (especially considering we say user to mean end users in the next sentence)
```suggestion
For inbound messages, SMS double opt-in is supported for all languages defined in the Subscription Group. This means you can define your auto-responses in different languages and Braze will send the auto-response associated with a specific language when a matching keyword is received.
```
